external/bcrypt/CMakeLists.txt

```diff
@@ -34,3 +34,16 @@ endif()
 if (OPENDAQ_ENABLE_TESTS)
     add_subdirectory(tests)
 endif()
+
+include(GNUInstallDirs)
+
+install(TARGETS ${EXTERNAL_LIB} EXPORT ${EXTERNAL_LIB}
+    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+)
+  
+install(
+    DIRECTORY "include/${EXTERNAL_LIB}"
+    DESTINATION include
```

Comment:
This should probably be `${CMAKE_INSTALL_INCLUDEDIR}/${EXTERNAL_LIB}`

Reply:
I will change this in the upcoming PR that targets main